lib/helpers.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * Extracts the measurement prefix from the configuration.
+ */
+const measurementPrefix = (config) => {
+
+    config = config || {};
+    const prefix = config.prefix;
+    let delimiter = config.prefixDelimiter;
+
+    let result = '';
+    if (Array.isArray(prefix)) {
+        if (typeof delimiter !== 'string') {
+            delimiter = '/';
+        }
+        result = `${prefix.join(delimiter)}${delimiter}`;
+    }
+    else if (typeof prefix === 'string') {
+        result = prefix;
+    }
+    return result;
```

Comment:
Should we escape the prefix using `formatTagKV()`? As per special characters of line protocol?

https://docs.influxdata.com/influxdb/v1.3/write_protocols/line_protocol_tutorial/

Reply:
The line protocol uses the formatters to escape the prefix + measurement name for each event:
```
const prefix = Helpers.measurementPrefix(config);
const finalEventName = Formatters.Measurement(`${prefix}${eventName}`);
```